src/js/module/toolbox/sidebar.js

```diff
@@ -1,58 +1,97 @@
-import underscore from 'underscore';
+import { chain, range, map } from 'underscore';
 import jquery from 'jquery';
-import { View } from 'core';
-import ToolboxViewObject from './object';
-import ToolboxCollection from './collection';
-import Const from 'const';
+import { View, Collection } from 'core';
+import ToolboxModel from '../plant/model';
```

Comment:
Do I correctly understand, that `ToolboxModel` here is `PlantModel` imported from `../plant/model.js`?